pages/bare_metal_cloud/dedicated_servers/getting-started-with-dedicated-server-eco/guide.en-gb.md

```diff
@@ -47,33 +47,33 @@ After choosing `Install from an OVHcloud template`{.action} you can select the o
 
 ![Operating selection](images/reinstalling-your-server-03.png){.thumbnail}
 
-If you need to modify the partioning scheme of your operating system, check the box "Customise the partition configuration" before clicking on `Next`{.action}.
+If you need to modify the partioning scheme of your operating system, check the box "SSH key configuration" before clicking on `Next`{.action}.
```

Comment:
Is this maybe a mistake? You did not edit this button in other languages and overall it does not make much sense.